<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSingleBandGrayRendererWidgetBase</class>
 <widget class="QWidget" name="QgsSingleBandGrayRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLineEdit" name="mMinLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mGrayBandLabel">
       <property name="text">
        <string>Gray band</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="mMaxLineEdit"/>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QComboBox" name="mGrayBandComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mContrastEnhancementLabel">
       <property name="text">
        <string>Contrast
enhancement</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QComboBox" name="mContrastEnhancementComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color gradient</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="mGradientComboBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="mMinLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="mMaxLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="mMinMaxContainerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mGrayBandComboBox</tabstop>
  <tabstop>mGradientComboBox</tabstop>
  <tabstop>mMinLineEdit</tabstop>
  <tabstop>mMaxLineEdit</tabstop>
  <tabstop>mContrastEnhancementComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
